troubleshooting/newer-hardware-troubleshooting.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: doc
+title: Troubleshooting newer hardware
+permalink: /doc/newer-hardware-troubleshooting/
+---
+
+Troubleshooting newer hardware
+==============================
+
+By default, the kernel that is installed in dom0 comes from the `kernel` package, which is an older Linux LTS kernel.
+For most cases this works fine since the Qubes OS developers backport fixes to this kernel, but for some newer hardware, you may run into issues.
```

Comment:
To be precise, Linux kernel developers does that - see https://www.kernel.org/category/releases.html